modules/homepage/src/main/frontend/src/themePage/Navbars/headerStyle.jsx

```diff
@@ -122,6 +122,10 @@ const headerStyle = theme => ({
   },
   searchResultSubjectIcon: {
     backgroundColor: "orange"
+  },
+  highlightedText: {
+    fontWeight: "bold",
+    backgroundColor: theme.palette.warning.light
```

Comment:
```suggestion
    backgroundColor: theme.palette.warning.light,
    padding: "1px 2px",
    borderRadius: "4px"
```
(just trying to make it look a bit smoother smoother)